.github/actions/build-evm-client/ethjs/action.yml

```
@@ -0,0 +1,36 @@
+name: 'Build EthereumJS monorepo'
+description: 'Builds the EthereumJS monorepo'
+inputs:
+  repo:
+    description: 'Source repository to use to build EthereumJS'
+    required: true
+    default: 'ethereumjs/ethereumjs-monorepo'
+  ref:
+    description: 'Reference to branch, commit, or tag to use to build EthereumJS'
+    required: true
+    default: 't8ntool'
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout EthereumJS monorepo
+      uses: actions/checkout@v4
+      with:
+        repository: ${{ inputs.repo }}
+        ref: ${{ inputs.ref }}
+        path: ethereumjs
+
+    - name: Setup node
+      uses: actions/setup-node@v4
+      with:
+        node-version: 18
+
+    - name: Build monorepo
+      run: |
```

```suggestion
      shell: bash
      run: |
```
Found during build in action: https://github.com/ethereum/execution-spec-tests/actions/runs/10492268620/job/29063368608